zerver/lib/soft_deactivation.py

```diff
@@ -212,7 +212,7 @@ def do_soft_deactivate_users(users: List[UserProfile]) -> List[UserProfile]:
 
     return users_soft_deactivated
 
-def maybe_catch_up_soft_deactivated_user(user_profile: UserProfile) -> Union[UserProfile, None]:
+def maybe_activate_soft_deactivated_user(user_profile: UserProfile) -> Union[UserProfile, None]:
```

Comment:
Maybe an even better name would be `reactivate_user_if_soft_deactivated`?

Reply:
Yes, that sounds better. Changing!